content/blog/2024/09/chunking-dsl/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Unlocking Efficient Data Processing with the Chunking DSL"
+date: 2024-09-06
```

Comment:
I think we can merge it as soon as ready. It should become public later on when this release date is reached.

Reply:
Thanks! We just have to trigger the website build on that date, right?

Reply:
Ah not sure.

From what I remember, last time, I was able to commit with a date in the future. I think the website was built. Then the article was not visible until the target date was reached. However, I don't remember if a website build was triggered by someone else.

Reply:
you can trigger the build right away. The article will appear on the date mentioned.

Reply:
Thanks, guys. I will merge this later today.